### ISSUE TITLE
Stealth no working as intended

### ISSUE THREAD

**Issue by chocochip95** [SunwellTracker/issues#2618]
Decription: Stealth makes you undetectable to enemies, with more points into Master of Deception, enemies should never see you

How it works: Get seen by enemies over 5 yards away at same level with 3/3 Master of deception

How it should work: Shouldn't be seen at all
Spells : http://db.darkwizard.pl/?spell=1784#modified-by  (stealth)
http://db.darkwizard.pl/?spell=13958 (MOD R1)
http://db.darkwizard.pl/?spell=13970 (MOd R2)
http://db.darkwizard.pl/?spell=13971 (MOD R3°

Source (you should point out proofs of your report, please give us some source): 
Basically you shouldn't be able to be seen this easily

**Comment by Aridam** [comment]
Can you provide screenshots/videos of how far you get seen by same-Level enemies in Stealth with MoD R3?

I wouldn't say MoD should make you _undetectable_ -- there are some concrete rules around Stealth detection. However, it's different VS Players and VS Mobs.

Needs some extensive research or straightforward proof for that, as I'm not sure myself.

More here:
https://wow.gamepedia.com/Stealth_(mechanic)

> The chance of detection increases as you get closer, but also depends on the viewing angle (stand right in front of an enemy and you can guarantee they'll see you), the relative levels, and any perception or subtlety modifiers.

> The closer you are to your target, the higher the probability that you will be detected. The chance of detection is also increased if you anywhere in a 180° arc in front of the target; it is always best to approach a target from behind. Remaining stationary does not make you harder to detect.

> Sutblety, like detection, increases 5 points with each level. When maxed, MoD is equivalent to 3 extra character levels.

So if you're Level 80 with MoD R3, it's like you're L83 while in Stealth, making detection harder, but not _impossible_ -- that's not enough for that to happen. Being in front also increases the chance to be detected (like standing right in front of something), while coming up from behind doesn't, but you can _still_ be detected if you get too close.

There's also this:

> In PvE, monsters will normally react when you are close to being detected. Typically, they will look up and either say something or make a noise.

Do also remember various Subtlety and Detection Items, Racials, Abilities, Pet Abilities etc. which affect one or the other in some way. In Mobs' case specifically, there are instead special Mobs like a few in Cities, who can see you in Stealth as if you aren't, but the lurking-behind-them rule still applies, so you can Distract them to face their back on you to avoid detection.

https://www.engadget.com/amp/2010/03/10/encrypted-text-stealth-mechanics-deep-dive/?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAI6gxgmK4xv2oH1M1wC3uZb_h1nLdJao-yVBIJOgQlFzd01D-It7ItsI2qamcXyPch7sx8RkuHyIqAoHoIcltGy0RnHxC7HW3mIUNVYUSSVY_OCBfQK6aZMfI5tDS-wbJrRV1wdmkOoLzevBe_zxjg72NgLi1uwjfazBpsH0LT2M#

This article was written at the start of March, 2010 -- that puts it around [v3.3.2](https://wow.gamepedia.com/Patch_3.3.2).

Scroll down to "How detection works".

> We can avoid all detection by staying behind our enemies, although every player has a small sphere of detection with about a 2 yard diameter. If you're in that 2 yard sphere, on any side, you'll be detected.

Given this, there isn't a mention of whether or not Mobs have this 2yd detection radius that players do, even though there wouldn't really be any need for you to be _that_ close to anything, considering Melee Range is 5yd.

> You're also at risk of detection if you're directly in front of your enemy, and at a smaller risk of detection if you're in their peripheral vision.

See the [picture below the paragraph](https://s.blogcdn.com/wow.joystiq.com/media/2010/03/dangerzone-1268204149.jpg).

> Enemies that are higher level than you have a higher-than-normal chance to detect you, like bosses or elites. Enemies that are below your level have a less-than-normal chance to detect you, though the 2 yard sphere will cause you to be detected no matter what. Once you're behind them, though, you're completely invisible.

The last part is a bit conflicting, given the 2yd radius and the lack of mention of whether or not it applies to Mobs as well.

Now, the below screenshots and descriptions in that article are very important, as they show the distance of Detection a Blood Elf L80 Priest has on a Human L80 Rogue with and without MoD R3 while in Stealth, from the front and back.

Forum post with comments dated around v3.2.2:
https://www.wowhead.com/forums&topic=129577#p1559024

This supports the concept of direct vision and peripheral vision, though it contradicts the article's pictured dangerzone in regards to being behind the Target.

>Actually, math says that the range where you absolutely cannot see a rogue is 90 degrees behind you, not 180.
